### DISCUSSION TITLE
What builds will be supported?

### DISCUSSION THREAD

**Discussion by carmen642** [RealMCoded/LunarRec#10] [Q&A] [answered]
What builds will be supported from 20XX-20XX(I hope it is up to 2021)

**Comment by RealMCoded** [comment, marked as answer]
Ignore my previous reply if you saw it, I thought this was the Rec.js repository instead of the LunarRec one.

LunarRec only supports 2017 (and 2016 if you enable the insecure mode). I might expand the projects scope to also work with some early 2018 builds but nothing later than September due to major API changes.

I would have to work 2021 support into a different server just due to how much work that is. And that's not even thinking about patching it, which is even harder.

TL;DR: LunarRec is a server for 2017 versions, 2021 support is not planned.
